homeassistant/components/switcher_kis/const.py

```diff
@@ -1,10 +1,15 @@
 """Constants for the Switcher integration."""
 
 DOMAIN = "switcher_kis"
-
+DATA_DISCOVERY = "discovery"
 DISCOVERY_TIME_SEC = 12
 
 SIGNAL_DEVICE_ADD = "switcher_device_add"
+CONF_USERNAME = "username"
+CONF_TOKEN = "token"
+
+COVER1_ID = "runner"
+COVER2_ID = "runner2"
```

Comment:
Is it going to be used anywhere in this PR? if it is for future device add it when you add the support for that device

Reply:
I need the COVER1_ID but removed the COVER2_ID